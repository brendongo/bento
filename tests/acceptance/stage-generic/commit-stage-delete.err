Running Bento checks...

⚠ No staged files to check.
◦ To check unstaged diffs, run $ bento check.
◦ To check a specific path, run $ bento check PATH.

✔ 0 findings in  s
