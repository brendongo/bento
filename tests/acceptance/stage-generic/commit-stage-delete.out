[master ] foo
 1 file changed, 1 deletion(-)
 delete mode 100644 foo.py
